edit/management.cattle.io.user.vue

```diff
@@ -110,28 +110,19 @@ export default {
         throw new Error(this.t('user.edit.credentials.username.exists'));
       }
 
-      const user = await this.$store.dispatch('management/create', {
-        type:               MANAGEMENT.USER,
-        metadata:           { generateName: `user-` },
+      const user = await this.$store.dispatch('rancher/create', {
+        type:               NORMAN.USER,
         description:        this.form.description,
-        displayName:        this.form.displayName,
         enabled:            true,
         mustChangePassword: this.form.password.userChangeOnLogin,
-        // password:           this.form.password.password,
-        username:           this.form.username,
+        name:               this.form.displayName,
+        password:           this.form.password.password,
```

Comment:
This is where the generated password is failing to be sent

Reply:
I couldn't reproduce the error, neither using the same browser or a different one.